Source/Scene/Cesium3DTileset.js

```diff
@@ -892,13 +879,8 @@ define([
 
     function raiseLoadProgressEvents(tileset, frameState) {
```

Comment:
Hm.. maybe this should be renamed to `raiseLoadProgressEvent` since there's only one now.